www/pages/pricing/index.tsx

```diff
@@ -12,12 +12,30 @@ import ReactMarkdown from 'react-markdown'
 import CTABanner from '~/components/CTABanner'
 
 import Solutions from 'data/Solutions.json'
+import { NextSeo } from 'next-seo'
 
 export default function IndexPage() {
   const { basePath } = useRouter()
 
+  const meta_title = 'Pricing | Supabase'
```

Comment:
title: Pricing & fees | Supabase
description: Explore Supabase fees and pricing information. Find our competitive pricing tiers, with no hidden pricing. We have generous free tiers for those getting started, and Pay As You Go for those scaling up.